src/main/java/seedu/address/logic/commands/ArchiveCommand.java

```diff
@@ -10,7 +10,7 @@
 import seedu.address.model.Model;
 import seedu.address.model.person.Person;
 
-public class ArchiveCommand extends RedoableCommand {
+public class ArchiveCommand extends Command {
 
```

Comment:
`ArchiveCommand` should probably be redoable. 
E.g. you addmodule once. Then you archive 3 person. If you undo, you will go back to the state of the addmodule, instantly unarchiving the 3 person.

Reply:
Yeah, we should keep with what @lawwm has refactored in his recent commits, (i.e. fixed redoable `ArchiveCommand`). It works well, and it also makes sense to be redoable.